<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="fig_show">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>5</y>
      <width>990</width>
      <height>990</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主要内容为:
1.对数据的描述性统计，主要通过散点图和直方图来进行判断
2.对数据的分布的判断，主要为正态分布(毕竟这个最具有参考性)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>
我从豆瓣爬取的Top250电影信息，
并尝试进行粗略的数据分析和理解
(由于时间限制，并没有进行深一步的数据建模)，
并将结果进行展示
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rank_mark">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>排名评分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>散点图:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>直方图:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>QQ图:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mark_number">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>评分评价人数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rank_number">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>排名评价人数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mark_number_hist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>评分与评价人数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="types_hist">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>影片种类</string>
    </property>
   </widget>
   <widget class="QPushButton" name="countries_hist">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>制作国家</string>
    </property>
   </widget>
   <widget class="QPushButton" name="languages_hist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>影片语言</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time_hist">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上映时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="long_hist">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>影片时长</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mark_QQ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>评分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Number_QQ">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>评价人数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Countries_QQ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>国家地区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Types_QQ">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>影片类型</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Languages_QQ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>影片语言</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Time_QQ">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上映时间</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
